variables.tf

```
@@ -176,6 +176,12 @@ variable "vpc_security_group_ids" {
   default     = null
 }
 
+variable "ipv6_allowed_for_dual_stack" {
```

```suggestion
variable "vpc_ipv6_allowed_for_dual_stack" {
```
Please rename it everywhere to be identical to other variable names.